crates/swc_ecma_loader/src/resolvers/node.rs

```diff
@@ -294,14 +309,17 @@ impl NodeModulesResolver {
         base_dir: &Path,
         target: &str,
     ) -> Result<Option<PathBuf>, Error> {
-        let mut path = Some(base_dir);
+        let absolute_path = to_absolute_path(base_dir).unwrap();
```

Comment:
This also can be an issue.
Please use `to_absolute_path(base_dir)?` instead

Reply:
4bcc044b013a76a1974e93506d7b1ece577cfd5a should fix the issue